### PR TITLE
NetworkUtils should be able to hold a free port for later usage #1109

### PR DESCRIPTION
**Pull request by ffang**
(cherry picked from commit b4282c88b91f5432e647696711c07e61812e7e16)

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: tests/itests-common/src/test/java/org/apache/camel/kafkaconnector/common/utils/NetworkUtils.java [hunk 1]
```diff
@@ -29,10 +29,11 @@
 import org.slf4j.LoggerFactory;
 
 public final class NetworkUtils {
-    public static final int  DEFAULT_STARTING_PORT = 49152;
+    
     public static final int  DEFAULT_ENDING_PORT = 65535;
     private static String hostname;
 
+    public static int  DEFAULT_STARTING_PORT = 49152;
```
Comment (tadayosi) [inline tests/itests-common/src/test/java/org/apache/camel/kafkaconnector/common/utils/NetworkUtils.java:36]: I'd keep `DEFAULT_STARTING_PORT` as `static final int` and declare another `static int startingPort` for actually using it in the code. There is also a style error that all uppercase variables need to be declared before lowercase variables.

**Review by tadayosi — CHANGES_REQUESTED** [review]
One small checkstyle issue. Please have a look at it.

File: tests/itests-common/src/test/java/org/apache/camel/kafkaconnector/common/utils/NetworkUtils.java (hunk 1 above)
Comment (ffang) [reply tests/itests-common/src/test/java/org/apache/camel/kafkaconnector/common/utils/NetworkUtils.java:36]: Thanks @tadayosi for the feedback, will revise accordingly

Comment (ffang) [reply tests/itests-common/src/test/java/org/apache/camel/kafkaconnector/common/utils/NetworkUtils.java]: Hi @tadayosi ,

Pushed new commits to address your feedback. 

Thanks!
Freeman

**Review by tadayosi — APPROVED** [review]
Looks awesome. Thanks!

**Review by orpiske — APPROVED** [review]
LGTM, thanks!!!

**Pull request merged**